### PR TITLE
test(onboarding):  12 18 and 24 seed phrase tests

### PR DESCRIPTION
**Pull request by OlufemiAdeOlusile**
### What does the PR do
This PR tests positive and negative flows for 12, 18, 24 seed phrase via Squish
https://github.com/status-im/status-desktop/issues/6274
https://github.com/status-im/status-desktop/issues/6275
https://github.com/status-im/status-desktop/issues/6276

### Affected areas

Depends on Icon change (include Object Name) via Status Q https://github.com/status-im/StatusQ/pull/761

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### Cool Spaceship Picture

<!-- optional but cool ->

### CODE REVIEW

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 1]
```diff
@@ -11,65 +11,191 @@
...
+
+        Examples:
+            | username |
+            | Athl     |
+            | Nervo    |
+            | Gra      |
+            | tester3@ |
+
+
+        Scenario Outline: User cannot sign up with wrong password format
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:36]: Identation

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 2]
```diff
@@ -11,65 +11,191 @@
...
+        Then the following ui-component mainWindow_Create_password_StatusBaseText is not enabled
+
+        Examples:
+            | username  | wrongpassword | password         |
+            | tester123 | Invalid34     | TesTEr16843/!@00 |
+            | tester124 | badP          | TesTEr16843/!@01 |
+            | tester124 | bad2!s        | TesTEr16843/!@01 |
+
+
+        Scenario Outline: User cannot finish Sign Up and Sign In process with wrong password
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:58]: Identation

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 3]
```diff
@@ -11,65 +11,191 @@
...
+        When user inputs the following <wrongpassword> with ui-component loginView_passwordInput
+        Then the following ui-component mainWindow_Finalise_Status_Password_Creation_StatusBaseText is not enabled
+
+        Examples:
+            | username  | wrongpassword        | password         |
+            | tester123 | Invalid34            | TesTEr16843/!@00 |
+            | tester123 | TesTEr16843/!@)      | TesTEr16843/!@01 |
+
+
+        Scenario: User signs up with imported 12 seed phrase
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:77]: Identation

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 4]
```diff
@@ -11,65 +11,191 @@
...
+            | seven     | 6          |
+            | budget    | 9          |
+            | denial    | 12         |
+            | body      | 15         |
+            | device    | 18         |
+
+        And user clicks on the following ui-component mainWindow_submitButton_StatusButton
+        When user signs up with username tester124 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user opens app settings screen
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:129]: Not sure why opening settings and wallet section is part of sign up suite

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 5]
```diff
@@ -11,65 +11,191 @@
...
+            | tester3@ |
+
+
+        Scenario Outline: User cannot sign up with wrong password format
+        Given A first time user lands on the status desktop and generates new key
+        When the user inputs username <username>
+
+        # Input wrong password format in both new password and confirmation input and verify create password button is not enabled
+        When user inputs the following <wrongpassword> with ui-component loginView_passwordInput
+        And user inputs the following <wrongpassword> with ui-component mainWindow_inputValue_StyledTextField
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:42]: Could the `ui-component` name be more specific? By reading this, I'm not sure which component are you referring and in which page are you.

**Review by noeliaSD — CHANGES_REQUESTED** [review]
I've tryied to run the scenarios related to seed phrases and I get the following test failures (same happens with `chatFlow` once the user lands into the app, after the onboarding process):

Screenshot 2022-07-11 at 10 09 00

https://user-images.githubusercontent.com/97019400/178218751-5601e3d2-08a8-491c-8532-c62d874ba36a.mov

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature (hunk 4 above)
Comment (noeliaSD) [reply test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:129]: Sorry, I don't have access to `TestRail`.. so, is it part of `SignUp` test case?

File: test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature [hunk 6]
```diff
@@ -11,65 +11,191 @@
...
+            | angle     | 16         |
+            | exhibit   | 20         |
+            | actress   | 24         |
+
+        And user clicks on the following ui-component mainWindow_submitButton_StatusButton
+        When user signs up with username tester124 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user opens app settings screen
+        And  the user activates wallet and opens the wallets section
+        Then the 24 seed phrase address is 0x28cf6770664821a51984daf5b9fb1b52e6538e4b displayed in the wallet
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature:169]: Not sure to undertand why this cannot be data-driven. Could you clarify more?

**Review by iurimatias — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]
With the new changes the scenario run correctly.

**Pull request merged**